copier.yml

```diff
@@ -68,8 +68,8 @@ command_line_interface:
     choices:
         - Click
         - Typer
-        - Argparse
-        - No command-line interface
+        # - Argparse
```

Comment:
I think I like have those two options. There are cases that it is useful to have a minimalist package. I do use `argparse` for some specific cases. There are also several cases of no need for CLI.

Reply:
Just like for pytest, I don't have a strong opinion right now, but argparse and no cli are not actually supported right now, so I just commented them out instead of deleting

Reply:
I think you misunderstood me. I think supporting argparse and no CLI is a good idea! 

But in this snapshot of the code, those options don't actually work right now. I think we _should_ implement them and add these back to the question form, but they are straight up broken right now